### PR TITLE
First half of the lab manual update

### PR DESCRIPTION
**Pull request by jennifermarsman**
I covered all of the bot setup and the LUIS model creation.  Tomorrow, I'll complete the lab manual with how to integrate calling LUIS and Azure Search into the bot.

### CODE REVIEW

File: LabManual.md [hunk 1]
```diff
@@ -205,3 +205,132 @@ Click **Search** to search for all documents.
...
+
+> Fun Aside: Notice that there is also an "Import App" next to the "New App" button on [the current page](https://www.luis.ai/applications).  After creating your LUIS application, you have the ability to export the entire app as JSON, and check it into source control.  This is a recommended best practice so you can version your LUIS models as you version your code.  An exported LUIS app may be re-imported using that "Import App" button.  If you fall behind during the lab and want to cheat, you can click the "Import App" button and import the [LUIS model from the Finished section](./Finished/LUIS/CognitiveServicesTutorialLuisModel.json).  
+
+From https://www.luis.ai/applications, click the "New App" button.  Give it a name (I chose "PictureBotLuisModel") and set the Culture to "English".  You can optionally provide a description.  Click the dropdown to select an endpoint key to use, and if the LUIS key that you created on the Azure portal at the beginning of this lab is there, select it.  Then click "Create".
```
Comment (noodlefrenzy) [inline LabManual.md:250]: Might want something in here about ensuring that you're logged into luis.ai using the same account you used to create the keys earlier.

File: LabManual.md [hunk 2]
```diff
@@ -205,3 +205,132 @@ Click **Search** to search for all documents.
...
+
+Now let's see how to create an entity.  When the user requests to search the pictures, they may specify what they are looking for.  Let's capture that in an entity.  
+
+Click on "Entities" in the left-hand column and then click "Add custom entity".  Give it an entity name "facet" and entity type "Simple".  Then click "Save".  
+
+![Add Facet Entity](./assets/AddFacetEntity.jpg) 
+
+Now click "Intents" in the left-hand sidebar and then click the yellow "Add Intent" button.  Give it an intent name of "SearchPics" and then click "Save".  
+
+Now let's add some sample utterances.  People might search for pictures in many ways.  Feel free to use some of the utterances below, and add your own wording for how you would ask a bot to search for pictures.
```
Comment (noodlefrenzy) [inline LabManual.md:278]: Include parenthetical small def'n of utterances. i.e. "sample utterances (simple sentences you think users might say that the bot should understand)"

**Review by noodlefrenzy — APPROVED** [review]
@jennifermarsman some very minor comments. Merging so we can have people start testing.

File: LabManual.md (hunk 1 above)
Comment (jennifermarsman) [reply LabManual.md:250]: Great point - this will be in the next checkin.

File: LabManual.md (hunk 2 above)
Comment (jennifermarsman) [reply LabManual.md:278]: Adding in next checkin

**Pull request merged**